back-end/tests/models/assignedBadges.test.js

```diff
@@ -0,0 +1,31 @@
+const {
+  createAssignedBadges,
+  createAdventurer,
+  createBadge,
+} = require("../seeds");
+const AssignedBadges = require("../../db/models/classProgress");
+
+let assignedBadges;
+let adventurer;
+let badge;
+
+describe("assigned badges model", () => {
+  beforeEach(async () => {
+    adventurer = await createAdventurer();
+    badge = await createBadge();
+    assignedBadges = await createAssignedBadges({
+      badge_id: badge.id,
+      adventurer_id: adventurer.id,
```

Comment:
There's indeed an `adventurer_id` column in that `assigned_badges` table, but the ERD in the readme says `user_id`. Might be worth updating the readme.

Reply:
yes, I got confused by that as well because I was using the ERD as a reference. I will go change it today.

Reply:
I don't see the ERD update in that PR. Did you plan to do it in another PR?

Reply:
The file itself has been updated, but it's a .png and it is name exactly the same so I don't know if there's a dif?

Reply:
![image](https://user-images.githubusercontent.com/53023071/82846869-f6341180-9eb8-11ea-8a3c-702490d6641f.png)

Reply:
it's binary file, and since you made a change to it, quite naturally, there will be a diff 😉